src/Padawan/Framework/Domain/Project/Persister.php

```diff
@@ -26,9 +26,10 @@ public function __construct(PathResolver $path, LoopInterface $loop)
     public function save(Project $project)
     {
         $this->checkForPadawanDir($project->getRootFolder());
-        return $this->fs->file(
-            $this->getProjectIndexFilePath($project->getRootFolder())
-        )->putContents($this->serialize($project));
+        return file_put_contents(
```

Comment:
`fs->file` is async operation, but file_put_contents always will be blocking. So won't we loose in perf?